### PR TITLE
Hotfix: release/7.1.1 (setup.py and package_data)

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Fixed:
- Moved all requirements to setup.py, and added all package_data there. Clean up unused files.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -4,17 +4,3 @@ include LICENSE
 include README.rst
 include requirements.txt
```
Comment (ashwini06) [inline MANIFEST.in:5]: `requirements.txt` is removed right?

File: setup.py [hunk 2]
```diff
@@ -20,11 +14,36 @@
...
+        "h5py>=3.1.0",
+        "PyPDF2>=1.26.0",
+        "markdown==3.3.3",
+        "cryptography<3.4",
+    ],
     packages=find_packages(),
     package_data={
-        "assets": ["scripts/*R", "sentieon_models/*model", "vcfanno/*.toml"],
-        "config": ["*.json"]
+        "": [
```
Comment (ashwini06) [inline setup.py:43]:
```suggestion
        "assets??": [
```

**Review by ashwini06 — COMMENTED** [review]
Few comments..

**Comment by hassanfa** [comment]
install succesful:
image

image

File: MANIFEST.in (hunk 1 above)
Comment (hassanfa) [reply MANIFEST.in]: yes. forgot to remove it from here.

File: setup.py (hunk 2 above)
Comment (hassanfa) [reply setup.py:43]: It'll include all files with those extensions. This way we won't forget to add them to package_data.

**Pull request merged**